js/presenters.js

```diff
@@ -0,0 +1,200 @@
+    name: 'Nanotte Ben Nana',
+    title: 'Professor of Economics at Moussa University',
+  },
+  {
+    description:
+      'Cythia von Geyin CEO is the author of "The reverse Flood" Ms von Geyin is the CEO of T34, the most important freight company of the World.',
+    altText: 'Photo Cythia von Geyin',
+    source: 'assets/images/speakers/speaker5.png',
+    name: 'Cythia von Geyin',
+    title: 'CEO of T34, Author of "The Reverse Flood"',
+  },
+  {
+    description:
+      'Lipo Mennendez is an anthropologist, Columnist, Author of "You Broke it, You Own It". Mr Mennendez is the advocate of the Yagaga Native People.',
+    altText: 'Photo Lipo Mennendez',
+    source: 'assets/images/speakers/speaker6.png',
+    name: 'Lipo Mennendez',
+    title: 'Columnist, Author and Yagaga People Advocate',
+  },
+];
+
+const speakers = document.createElement('section');
+speakers.setAttribute('class', 'presenters');
+
+const feature = document.createElement('h2');
+feature.setAttribute('class', 'speakersH1');
+feature.textContent = 'Featured Speakers';
+speakers.appendChild(feature);
+
+const hLine = document.createElement('hr');
+speakers.appendChild(hLine);
+
+const speakerWrapper = document.createElement('div');
+speakerWrapper.setAttribute('class', 'wrapper');
+speakers.appendChild(speakerWrapper);
+
+const parentNode = document.querySelector('body');
+const existingNode1 = document.querySelector('.partners');
+const existingNode = document.querySelector('.more');
+
+if ($(window).width() > 768) {
+  for (let num = zero; num < speakersData.length; num += one) {
+    const speakerItem = document.createElement('div');
+    speakerItem.setAttribute('class', 'item');
+
+    const poster = document.createElement('img');
+    poster.setAttribute('class', 'posterImage');
+    poster.setAttribute('src', speakersData[num].source);
+    poster.setAttribute('alt', speakersData[num].altText);
+    speakerItem.appendChild(poster);
+
+    const details = document.createElement('div');
+    details.className = 'detail';
+
+    const theName = document.createElement('p');
+    theName.setAttribute('class', 'speakerName');
+    theName.textContent = speakersData[num].name;
+    details.appendChild(theName);
+
+    const speakerTitle = document.createElement('p');
+    speakerTitle.setAttribute('class', 'titleOfSpeaker');
+    speakerTitle.textContent = speakersData[num].title;
+    details.appendChild(speakerTitle);
+
+    const line = document.createElement('hr');
+    line.setAttribute('class', 'hrLine');
+    details.appendChild(line);
+
+    const info = document.createElement('p');
+    info.setAttribute('class', 'descriptions');
+    info.textContent = speakersData[num].description;
+    details.appendChild(info);
+
+    speakerItem.appendChild(details);
+
+    speakerWrapper.appendChild(speakerItem);
+
+    existingNode1.parentNode.insertBefore(speakers, existingNode1);
+  }
+}
+
+if ($(window).width() <= 768) {
+  // do your stuff
```

Comment:
- [ ] Your JavaScript code looks good 👏. Please avoid leaving unnecessary comments in your code. it makes your code not very clean. comments should be to add information about complex parts of your code that others might want to read. kindly remove this line of code

Reply:
Required change made